app/controllers/messages_controller.rb

```diff
@@ -2,84 +2,94 @@ class MessagesController < ApplicationController
 
   before_filter :require_user
   before_filter :require_owner, only: [ :show, :destroy ]
-  
+
   def new
-	  #create the message
-	  @message = Message.new
-	  @title = "New message"
-      @users = User.all
-	  # ideally, we would kick out the current_user however, this generates crashes when there are only two users (function "map" doesn't work on a single object)
-      if params[:message] 
-         @answering = Message.find_by_id(params[:message])
-         @allowed = true
-         if @answering.from_id != current_user.id and @answering.to_id != current_user.id 
-             @allowed = false
-         end
+    #create the message
+    @message = Message.new
+    @title = "New message"
+    @users = User.all
+    # ideally, we would kick out the current_user however, this generates crashes when there are only two users (function "map" doesn't work on a single object)
+    if params[:message] 
+      @answering = Message.find_by_id(params[:message])
+      @allowed = true
+      if @answering.from_id != current_user.id and @answering.to_id != current_user.id 
+        @allowed = false
       end
-	  respond_to do |format|
-		  format.html
-		  format.xml { render :xml => @message }
-	  end
+    end
+    respond_to do |format|
+      format.html
+      format.xml { render :xml => @message }
+    end
   end
 
   def create
-	  @message = Message.new(params[:message])
-	  @message.user_id = current_user.id
-	  @message.user_has_seen = true
-	  @message.from_id = current_user.id
-      @message.sent = true
+    @message = Message.new(message_params)
+    @message.user_id = current_user.id
+    @message.user_has_seen = true
+    @message.from_id = current_user.id
+    @message.sent = true
 
-	  if params[:user] != nil
-		  @message.to_id = params[:user][:id]
-	  end
+    if params[:user] != nil
+      @message.to_id = params[:user][:id]
+    end
 
-	  if @message.save and @message.send_message(@message.from_id, @message.to_id)
-		  flash[:notice] = "Message sent!"
-		  redirect_to "/users/" + current_user.id.to_s + "#messages"
-	  else
-		  respond_to do |format|
-			  format.html { render :action => "new" }
-			  format.xml { render :xml => @message.errors, :status => :unprocessable_entity }
-		  end
-	  end
+    if @message.save and @message.send_message(@message.from_id, @message.to_id)
+      flash[:notice] = "Message sent!"
+      redirect_to "/users/" + current_user.id.to_s + "#messages"
+    else
+      respond_to do |format|
+        format.html { render :action => "new" }
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.